### DISCUSSION TITLE
`<pre>` tag is not adapting the column width in bootstrap

### DISCUSSION THREAD

**Discussion by thelovekesh** [twbs/bootstrap#33640] [Q&A]
I am using `Bootstrap 5` to style my page. When I am using the `<pre>...</pre>` tag inside the `class="col"` then it is not adapting the width of the column instead of that it is increasing the width of the whole column. But when I am using the `class="col-lg-8"` it is working fine.

### HTML CODE
```html
<div class="col">
    <pre>
          <code>
             ...
          </code>
    </pre>
</div>
```

### With `<pre>` tag and `class="col"`
![image](https://user-images.githubusercontent.com/54371619/114841491-8773e580-9df5-11eb-8a72-7268038bd731.png)

### With `<pre>` tag and `class="col-lg-8"`
![image](https://user-images.githubusercontent.com/54371619/114841577-a1152d00-9df5-11eb-97e3-e5562dc87423.png)

### Without `<pre>` tag
![image](https://user-images.githubusercontent.com/54371619/114841642-b4c09380-9df5-11eb-9dce-9df52560d1fe.png)

**Comment by thelovekesh** [comment]
**_Check it live here_**
https://thelovekesh.github.io/cbtheme-debug/blog/

**Comment by alpadev** [comment]
This is somewhat expected. See https://stackoverflow.com/questions/36247140/why-dont-flex-items-shrink-past-content-size#answers-header

**Comment by dvmuccillo** [comment]
Hi @thelovekesh ,
i solved this problem on a couple of pages using
`white-space: pre-wrap;`, and it seems to work on
your page (tested with Chrome DevTools).

![image](https://user-images.githubusercontent.com/6137203/114906949-f6027480-9df0-11eb-9654-87f445e67a17.png)
